src/main/java/org/folio/rest/impl/PostOrdersHelper.java

```diff
@@ -32,11 +35,13 @@
   private static final long PO_NUMBER_EPOCH = 1535774400000L;
 
   private final ValidationHelper validationHelper;
+  private final PutOrdersByIdHelper putOrderHelper;
 
   public PostOrdersHelper(HttpClientInterface httpClient, Map<String, String> okapiHeaders,
       Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler, Context ctx, String lang) {
     super(httpClient, okapiHeaders, asyncResultHandler, ctx, lang);
     this.validationHelper = new ValidationHelper(httpClient, okapiHeaders, asyncResultHandler, ctx, lang);
+    this.putOrderHelper = new PutOrdersByIdHelper(okapiHeaders, asyncResultHandler, ctx, lang);
```

Comment:
It feels a little weird to have a putOrderHelper as a member of the postOrderHelper.  I think the approach we've tried to take in situations like this is to split code that's shared between helpers in a common/shared location, e.g. HelperUtils rather than having the helpers reference one another.